bundles/org.openhab.binding.webexteams/README.md

```diff
@@ -0,0 +1,83 @@
+# WebexTeams Binding
+
+The Webex Team binding allows to send messages to [Webex Teams](https://web.webex.com/) users and rooms through a number of actions.
+
+Messages can use markdown syntax, and attachments are supported.
+
+## Supported Things
+
+- `account`: A Webex Teams account
+
+## Discovery
+
+No Things are being discovered by this binding.
+
+
+## Thing Configuration
+
+Webex Teams supports two main types of app integration:
+
+* Bot: a separate identity that can be used to communicate with people and rooms.
+* Integration: OAuth integration that allows the binding to act on a persons behalf.
+
+Both of these integrations must be first configured on the [Webex Developers](https://developer.webex.com/) website.
+
+To use the bot identity, only configure the `token` (Authentication token).
+
+To use the OAuth interation, configure: client id, client secret, auth code and redirect URL.  The auth code is obtained
```

Comment:
Can you put senstense on a single line to make changes more readable when updates to this file are made in the future.

Reply:
Done.